pages/club/[club]/index.js

```diff
@@ -49,28 +51,42 @@ const Club = ({ query, userInfo, favorites, updateFavorites, subscriptions, upda
   const { image_url: image } = club
 
   return (
-    <WideContainer>
-      <Flex>
-        {image && <Image src={image} />}
-        <Header
-          club={club}
-          userInfo={userInfo}
-          favorites={favorites}
-          updateFavorites={updateFavorites}
-          subscriptions={subscriptions}
-          updateSubscriptions={updateSubscriptions}
-          style={{ flex: 1 }}
-        />
-      </Flex>
+    <WideContainer background={`${SNOW}`} fullHeight={true}>
```

Comment:
You should be able to do just `background={SNOW}` instead of ``background={`${SNOW}`}``.